TreeAPI.js

```diff
@@ -100,12 +263,15 @@ WikiTreeAPI.getPerson = function(id,fields) {
 WikiTreeAPI.postToAPI = function(postData) {
 	var API_URL = 'https://api.wikitree.com/api.php';
 
+	// let f = [];
+	// if (postData.fields) {f = postData.fields.split(',')}
+	// condLog(`postToAPI: ${postData.action} ${postData.key}`, f)
 	var ajax = $.ajax({
```

Comment:
It's not your code, but it would be nice to get rid of jquery dependency.

JS natively offers `fetch API` to send requests

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#supplying_request_options

Reply:
He he, I know next to nothing of either, but will do some reading and see what I can come up with :)